fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/FabricLootTable.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.loot.v1;
+
+import java.util.List;
+
+import net.minecraft.loot.LootPool;
+import net.minecraft.loot.LootTable;
+import net.minecraft.loot.context.LootContextType;
+import net.minecraft.loot.function.LootFunction;
+
+/**
+ * Accessor methods for loot table properties.
+ * This interface is automatically implemented on all {@link LootTable} instances.
+ */
+public interface FabricLootTable {
```

Comment:
Ideally this interface should need to be exposed in API, a helper class with static `getBlah` methods would suffice here and be more resilient to api changes.

Reply:
I think you mean this interface *shouldn't* be exposed in the API? For that, I can make a `LootTables` class (prefixed with Fabric if needed) with a bunch of getters, and maybe do the same to loot pools too.